### ISSUE TITLE
install.python-poetry.org creates poetry command wrapper in wrong directory on macOS according to documentation

### ISSUE THREAD

**Issue by hutchisr** [python-poetry/install.python-poetry.org#30]
- [x] I am on the [latest](https://github.com/python-poetry/poetry/releases/latest) Poetry version.
- [x] I have searched the [issues](https://github.com/python-poetry/poetry/issues) of this repo and believe that this is not a duplicate.

- **OS version and name**: macOS 12.5.1 (arm64)
- **Poetry version**: 1.2.0
- **Python version**: 3.10.6_1 (Homebrew)
  (install with system python3 fails entirely) 

## Issue

According to the [documentation](https://python-poetry.org/docs/), the new poetry installation script should install the `poetry` command to `~/Library/Application Support/pypoetry/bin/poetry` with a wrapper in `$HOME/.local/bin` however neither of these locations get installed.

Instead, `poetry` gets installed to `~/Library/Application Support/pypoetry/venv/bin/poetry` with the wrapper put in the much less convenient location `~/Library/Python/3.x/bin` (with x being the minor python version, eg `~/Library/Python/3.10/bin`).

I assume the former location is just a typo but sticking the wrapper in this undocumented python version-specific seems undesirable and probably a bug with the installer.

**Comment by neersighted** [comment]
Looks like a couple things mixed together -- the venv directory got omitted from the docs by mistake and should be added, but that is expected behavior.

When you install Poetry, what does it print out? I suspect that this might be because of an undocumented change in Python 3.10 causing `site.getuserbase()` to return something different -- the full installer log plus `site.getuserbase()` from the Python you are using would be very helpful in figuring out what is going on.

**Comment by hutchisr** [comment]
Output of the installer is
```
$ curl -sSL https://install.python-poetry.org | python3 -

Retrieving Poetry metadata

# Welcome to Poetry!

This will download and install the latest version of Poetry,
a dependency and package manager for Python.

It will add the `poetry` command to Poetry's bin directory, located at:

/Users/<uername>/Library/Python/3.10/bin

You can uninstall at any time by executing this script with the --uninstall option,
and these changes will be reverted.

Installing Poetry (1.2.0): Done

Poetry (1.2.0) is installed now. Great!

To get started you need Poetry's bin directory (/Users/<uername>/Library/Python/3.10/bin) in your `PATH`
environment variable.

Add `export PATH="/Users/<uername>/Library/Python/3.10/bin:$PATH"` to your shell configuration file.

Alternatively, you can call Poetry explicitly with `/Users/<uername>/Library/Python/3.10/bin/poetry`.

You can test that everything is set up by executing:

`poetry --version`
```

and 

```
$ python3 -c 'import site; print(site.getuserbase())'
/Users/<username>/Library/Python/3.10
```

I get the same results for both commands using python3.9 as well (though with the directory being 3.9 instead of 3.10 obviously).

**Comment by neersighted** [comment]
This should be resolved with the merge of #36 -- looks like we forgot to account for the fact that `site.getuserbase()` has different output when Python is built as a macOS framework. Between that PR, and the Poetry docs PR, this should be fully resolved.
